lib/application/task-cache-hit.js

```diff
@@ -20,9 +20,13 @@ const getCacheHit = async (connectionString) => {
 const logCacheHits = async () => {
   const event = 'cache-hit-ratio';
   for (const scalingoApp of SCALINGO_APPS) {
-    const connectionString = await databaseStatsRepository.getDbConnectionString(scalingoApp);
-    const cacheHit = await getCacheHit(connectionString);
-    logger.info({ event, app: scalingoApp, data: { cacheHit } });
+    try {
+      const connectionString = await databaseStatsRepository.getDbConnectionString(scalingoApp);
+      const cacheHit = await getCacheHit(connectionString);
+      logger.info({ event, app: scalingoApp, data: { cacheHit } });
+    } catch (error) {
+      logger.error({ task: 'cache-hit', message: error.message.slice(0, 1000), error: error.response.data.error });
```

Comment:
Extraire dans une constante

```suggestion
      const STACKTRACE_EXTRACT_LENGTH = 1000;
      const event = { task: 'cache-hit', message: error.message.slice(0, STACKTRACE_EXTRACT_LENGTH), error: error.response.data.error 
      logger.error(event);
```